notify.go

```diff
@@ -30,13 +30,18 @@ type Notifier struct {
 
 // Notification contains all of the fields and settings that a notification can have
 type Notification struct {
-	Title        string
-	SubTitle     string
-	ContentImage string
-	Message      string
-	CloseText    string
-	Actions      []string
-	timeout      *int
+	Sender        string
+	AppIcon       string
```

Comment:
Were you able to get `AppIcon` to work? I have tried to set it in my example just like I do `ContentImage` but I don't get image. I'm on Ventura so perhaps your older version is working (from memory you mentioned you were using an older version in a VM in your issue)

Reply:
Yes  I used absolute path to a PNG file ad it work fine.

![image-1.png](https://user-images.githubusercontent.com/8627125/211923098-0585373d-b4b1-4b14-b475-f2b3a2521091.png)

Reply:
Hmmm interesting. I thought that image you have  there in the notification was the content image and the tiny item image was the app icon.

Reply:
For me the icon is on the left and the content image is on the right.
![Screenshot from 2023-01-14 15-58-17](https://user-images.githubusercontent.com/8627125/212478349-cb22df87-a186-4915-9084-ba298df6dd8a.png)

Reply:
Oh wow. That’s not the behaviour I see on Monterey or Ventura. 

And you’re setting the absolute path to both properties in the same way (ie you can put the same string path in both properties and it works for both)?

Reply:
Yes, I'm going to make a Ventura VM and test it too, when I have time.